contracts/test/v0.7/UpkeepRegistrationRequests.test.ts

```diff
@@ -29,7 +29,8 @@ before(async () => {
   mockV3AggregatorFactory = (await ethers.getContractFactory(
     'src/v0.7/tests/MockV3Aggregator.sol:MockV3Aggregator',
   )) as unknown as MockV3AggregatorFactory
-  keeperRegistryFactory = await ethers.getContractFactory('KeeperRegistry')
+  // @ts-ignore bug in autogen file
```

Comment:
does using the full import path resolve the TS bug? example on line 30.

Reply:
unfortunately no, this is a hardhat issue with naming. If I put 'KeeperRegistry11' then typechecker is happy but it fails while running

Reply:
Essentially solc is able to handle the '_', but hardhat can't.